### PR TITLE
Feature/app setup: Data fetch implementation

### PR DESCRIPTION
**Pull request by Diegodsha**
- Use Webpack
- Divide code into modules
- Use ES6 Syntax
- Use Open Weather API to fetch data
- Deployed to Netlify

### CODE REVIEW

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Invalid Code Review Request
Friendly reminder: You should use Github/Git Flow in this project.
Please take a step back and read this [lesson](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/) and do watch the video.
Then create a brand new repo with the correct setup and feature branch (with DESCRIPTIVE name) and copy your code there or close this pull request and create another one with the correct feature branch.
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Comment by Diegodsha** [comment]
Hello @SajjadAhmad14  can you be more specific with in which manner I'm not using git flow? (I think I'm using it correctly since I have main, develop and a feature branch) thanks for your time.

File: src/fetch.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const moment = require('moment');
```
Comment (SajjadAhmad14) [inline src/fetch.js]: Please use import.

File: src/fetch.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+const sky = document.querySelector('.sky');
+const humidity = document.querySelector('.humidity-percentage');
+const feelsLike = document.querySelector('.feels-like-deg');
+const windSpeed = document.querySelector('.wind-speed-text');
+const maxTemp = document.querySelector('.max-temp-text');
+const visibility = document.querySelector('.visibility-text');
+const pressure = document.querySelector('.hpa-text');
+const icon = document.querySelector('.weather-icon');
+
+async function getWeatherData(url = '') {
```
Comment (SajjadAhmad14) [inline src/fetch.js]: Use the arrow function here.

File: src/fetch.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      if (sky.innerText.includes('clouds')) {
+        icon.innerHTML = '<i class="fas fa-cloud-sun"></i>';
+      } else if (sky.innerText.includes('rain')) {
+        icon.innerHTML = '<i class="fas fa-cloud-rain"></i>';
+      } else if (sky.innerText.includes('clear')) {
+        icon.innerHTML = '<i class="fas fa-sun"></i>';
+      }
+    })
+    .catch((err) => {
+      console.log(err, 'failure-check-your-code');
```
Comment (SajjadAhmad14) [inline src/fetch.js]: Do not use `console.log`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
# Changes Required :recycle:
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hello, you have been doing a tremendous job so far :ok_hand:
## To Highlight 🎉
- [ ] You have mastered Git flow.
- [ ] PR title and summary are descriptive.
- [ ] README is professional and includes all the info to run the project locally.

## To Improve 🙏
- [x] App should have a toggle button to convert Celsius to Fahrenheit or vice versa.
- [x] App should change its appearance based on the received data. Please see this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png).
- [x] Please do not use `console.log`. It is anti performance and it can make app secret data open to the browser.
- [x] Please use ES6 syntax. Use arrow function instead of function keyword also use import rather than require.
- [ ] See inline comments for refference.
# Optional
- [ ] App may fetch the current geolocation of the user when it is first accessed and fetches the weather from that location

### Happy Coding ☺️

## If you have any questions you can comment here in this PR or can always connect with me on slack ( @Sajjad Ahmad ).

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔**

Hello @Diegodsha 
Impressive UI
Great work making the requested changes.

You can merge👏👏

Happy coding💻💻💻

**Pull request merged**